lib/transaction.js

```diff
@@ -1,7 +1,7 @@
 const { apiRequest } = require('./apiRequest');
 
-async function getTransactionListByUser() {
-  const apiResponse = await apiRequest('GET', '/transaction/find-by-user', '?p=1', null);
+async function getTransactionListByUser(param) {
```

Comment:
same as above for param

https://developer.mozilla.org/en-US/docs/Web/JavaScript/Reference/Functions/Default_parameters